__tests__/app.test.js

```diff
@@ -269,4 +269,130 @@ describe("POST /api/reviews/:review_id/comments", () => {
          })
    });
    
+});
+
+describe.only("PATCH /api/reviews/:review_id", () => {
+  test("Get status 200 response and an object containing the updated review with votes added on", () => {
+  const newVotes = {
+        inc_votes: 20
+      };
+      return request(app)
+        .patch("/api/reviews/1")
+        .send(newVotes)
+        .expect(200)
+        .then((response) => {
+          expect(response.body.review[0].review_id).toBe(1);
+          expect(response.body.review[0].title).toBe("Agricola");
```

Comment:
remove the .onlys before submitting code. I'd ussually ask for a resubmit, but since this is the last push, I'll take it on good faith that your other test are running... 😄